BigImageViewer/src/main/java/com/github/piasy/biv/view/ImageCycleCallback.java

```diff
@@ -0,0 +1,10 @@
+package com.github.piasy.biv.view;
+
+import androidx.annotation.UiThread;
+
+@UiThread
+public interface ImageCycleCallback {
```

Comment:
This callback is good, but its name is confusing, can we have a better name for it?

Reply:
Again, very bad with naming in general?
DelayedCycleCallback, DelayedImageCallback?

Reply:
Done